WalletWasabi.Tests/WalletWasabi.Tests.csproj

```diff
@@ -22,6 +22,11 @@
 			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
 			<PrivateAssets>all</PrivateAssets>
 		</PackageReference>
+		<PackageReference Include="FluentAssertions" Version="6.10.0" />
+		<PackageReference Include="FluentAssertions.Analyzers" Version="0.17.2">
+		  <PrivateAssets>all</PrivateAssets>
+		  <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
```

Comment:
@molnard 
Is it ok to introduce new packages on Tests level? 

Basically, with it you can say:
```c#
sut.TopSuggestions.Count.Should().Be(expectedSuggestionsCount);
```

Instead of:
```c#
Assert.Equal(sut.TopSuggestions.Count, expectedSuggestionsCount);
```